pgcontents/query.py

```diff
@@ -549,7 +550,8 @@ def save_file(db, user_id, path, content, encrypt_func, max_size_bytes):
     return res
 
 
-def generate_files(engine, crypto_factory, min_dt=None, max_dt=None):
+def generate_files(engine, crypto_factory, min_dt=None, max_dt=None,
+                   logger=None):
```

Comment:
This is a function to be used from outside pgcontents, so the idea is that the user may pass a logger if they wish.